tests/OpenTok/OpentokTestCase.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Opentok;
+
+use \Firebase\JWT\JWT;
+
+class OpenTokTestCase extends \PHPUnit_Framework_TestCase {
+  protected $API_KEY;
+  protected $API_SECRET;
+
+  public function validateJwt($token) {
```

Comment:
i'm not sure this strategy works because the tests are failing.

also, i can see you refactored in this way to work around the protected access on the `$API_KEY` and `$API_SECRET` variables. if that's the strategy you want to go about, theoretically all the functions in `TestHelpers` could be methods inside this base class and you could eliminate that file. i think it would be messy to have both.

but i actually prefer the way its been done in the `TestHelpers` class. each function in that class is static, so it doesn't depend on any state (which you should always strive to minimize in a test) like instance variables. each of them are pure functions. if you follow that strategy, you just change the method signature to include the API Key and API Secret, and it works just as well. for example: `pubic function validateJwt($token, $apiKey, $apiSecret)`. making a whole new file/class to get around 2 parameters here and losing out on the pure function seems like a bad tradeoff.

lastly, for naming, knowing that eventually we will have client tokens that are in the JWT format with slightly different validation logic, i would go with something that can differentiate. maybe `validateOpenTokAuthHeader()`?